systemd/systemd.go

```diff
@@ -323,23 +324,31 @@ func IsTimeout(err error) bool {
 
 const myFmt = "2006-01-02T15:04:05.000000Z07:00"
 
+func (l Log) Time() (time.Time, error) {
+	sus, ok := l["__REALTIME_TIMESTAMP"]
```

Comment:
answer: because it's only used here and in tests, so making it a constant would win us very little beyond the complexity